lib/src/get_annotated_tasks.dart

```diff
@@ -0,0 +1,55 @@
+
```

Comment:
Renamed `addAnnotatedTasks` to `_discoverTasks`, left `getAnnotatedTasks` et al. as is.